s30566-DanielMakowski/zajecia3/zadanie2.txt

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+#Napisz skrypt, który:
+#
+#sprawdzi, czy istnieje plik config.cfg w bieżącym katalogu.
+#
+#Jeśli taki plik istnieje, to niech go wykona.
+#
+#Jeśli nie istnieje, to niech sprawdzi, czy podano argument i wykona plik o nazwie takiej jak wartość argumentu (parametru) skryptu.
+#
+#Jeśli żadne z powyższych nie nastąpiło, to niech wyświetli komunikat o błędzie.
+
+one=$1
+
+if [ $(find . -maxdepth 1 -mindepth 1 -name config.cfg -type f) ]; then
+	./config.cfg
+elif [[ -n $one ]]; then
+	if [ $(find . -maxdepth 1 -mindepth 1 -name "$one" -type f) ]; then
```

Comment:
Proszę to uprościć. Tu trzeba skorzystać z operatora łączenia warunków logicznych.